pom.xml

```diff
@@ -115,7 +115,7 @@
     <pluginRepositories>
         <pluginRepository>
             <id>confluent</id>
-            <url>http://packages.confluent.io/maven/</url>
+            <url>https://packages.confluent.io/maven/</url>
```

Comment:
Can we target these changes in another PR? Looks like this change can be ported as old as 1.1.x. So we can raise a PR to that branch and forward merge it. WDYT?

Reply:
@mukkachaitanya The PR jenkins build was showing a red cross w.r.t "Code scanning"/CodeQL and so I had made that pom commit to make the build Green. I see your point though. Does the PR merge allow to force/bypass CodeQL check and is that advisable? If so I can revert the pom change from this PR and track that separately against 1.1.x as you pointed.

Reply:
> I see your point though. Does the PR merge allow to force/bypass CodeQL check and is that advisable?

I suppose we can. Another way to go about it is to create the PR to 1.1.x first and merge it through and then merge this PR.

Reply:
@mukkachaitanya I liked the later approach, thanks for the suggestion. Let me first do the pom PR alone and pint merge same, post that merge this current PR.

Reply:
@mukkachaitanya Raised [PR-229](https://github.com/confluentinc/kafka-connect-bigquery/pull/229). Please approve.

Reply:
@mukkachaitanya I have pint merged PR-229. Can you please approve current PR? Thanks!